src/test/fixtures/composing/src/main/kotlin/com/example/foo/Hello.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.foo
+
+abstract class Hello : com.example.foo.psi.FooGreetingStmt {
+  fun hello() {
+    val helloStmt: com.example.psi.RootHelloStmt = helloStmt
```

Comment:
This line is the interesting compiling test: Hello extends `greeting_stmt`, defined in `root.bnf`, but overwritten in `foo.bnf`. This overwritten rule has a child rule `hello_stmt_internal`, which references to a not available rule `hello_stmt`. With this PR it is finally possible to get the available rule.